navigation/waypoint_manager/scripts/waypoint_manager.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+from vortex_msgs.srv import Waypoint
+from nav_msgs.msg import Path
+from geometry_msgs.msg import PoseStamped, Point
+
+
+class WaypointManager(Node):
+    """
+    Nodes created:
+        WaypointManager
+    Subscribes to:
+    Publishes to:
+    """
+    
+    def __init__(self):
+        """
+        Initializes WaypointManager class.
+
+        Initializes node, sets up action client, services, and publishers.
+        """
+        super().__init__("WaypointManager")
+
+        self.waypoint_list = []
```

Comment:
Following the waypoint format discussion, this internal state can be maintained as a list of a `Waypoint2D` or something, like:

```python
@dataclass(frozen=True)
class Waypoint2D:
    north: float
    east: float
```
which f.ex. allows you to write
```python
self.waypoint_list.append(Waypoint2D(north=x, east=y))
```
instead